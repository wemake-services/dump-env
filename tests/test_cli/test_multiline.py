def test_simple_multiline(monkeypatch, delegator, snapshot):
    """Check that cli works with multiline inputs"""
    monkeypatch.setenv('MULTILINE_VALUE', '1\n2\n3')

    variables = delegator('dump-env -p MULTILINE_')
    assert variables == snapshot
