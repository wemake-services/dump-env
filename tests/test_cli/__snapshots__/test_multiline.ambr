# serializer version: 1
# name: test_simple_multiline
  '''
  VALUE="1
  2
  3"
  
  '''
# ---
